Applications/YOLO/jni/det_dataloader.cpp

```diff
@@ -0,0 +1,153 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2023 Seungbaek Hong <sb92.hong@samsung.com>
+ *
+ * @file   det_dataloader.h
+ * @date   22 March 2023
+ * @brief  dataloader for object detection dataset
+ * @see    https://github.com/nnstreamer/nntrainer
+ * @author Seungbaek Hong <sb92.hong@samsung.com>
+ * @bug    No known bugs except for NYI items
+ */
+
+#include "det_dataloader.h"
+
+#include <cstring>
+#include <filesystem>
+#include <fstream>
+#include <iostream>
+#include <nntrainer_error.h>
+#include <random>
+#include <sstream>
+
+namespace nntrainer::util {
+
+DirDataLoader::DirDataLoader(const char *directory_, unsigned int max_num_label,
+                             unsigned int c, unsigned int w, unsigned int h,
```

Comment:
It might be better to describe saying it supports bmp only for now.

Reply:
I added it, thanks.